Packs/AbuseDB/pack_meta.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "AbuseIPDB",
+    "description": "Central repository to report and identify IP addresses that have been associated with malicious activity online. Check the Detailed Information section for more information on how to configure the integration.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```

Comment:
ServerMInVersion should be server version and not pack version

Reply:
it is the server version.. there is no `fromversion`